packages/light-apps/public/index.html

```diff
@@ -1,16 +1,34 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="theme-color" content="#000000">
-    <!--
+
+<head>
+  <meta charset="utf-8">
+  <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta name="theme-color" content="#000000">
+  <!-- These CSP are for prod. For dev, CSP are set inside @electron-app -->
+  <meta http-equiv="Content-Security-Policy" content="
+    block-all-mixed-content;
+    child-src 'none';
+    connect-src http: ws:;
```

Comment:
The `http` here is not needed in prod, and _should_ be overwritten in the `onHeadersReceived`. However, somehow it doesn't work: https://stackoverflow.com/questions/51969512/define-csp-http-header-in-electron-app#answer-52243138

So we can:
- either add `http` in prod, which I believe doesn't introduce a large attack surface, and allow CRA hot reloading
- or remove `http` in prod, but then CRA hot reloading breaks.